src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -387,6 +381,22 @@ private void DebugVerifyNoErrors(ConflictResolution conflictResolution, IEnumera
                 // as the parent (yes I know, this is a simplification).
                 if (_documentIdOfRenameSymbolDeclaration.ProjectId == projectId)
                 {
+                    foreach (var documentId in allDocumentIdsInProject.Where(d => !documentIds.Contains(d)))
```

Comment:
Will add a comment explaining why we're doing this. The explanation is in the PR description for now.